history-of-quakers/modernized/22-chapter-20.adoc

```diff
@@ -1042,7 +1154,8 @@ Ought not therefore all Christians likewise to have the New Testament,
 that makes mention of Christ and his apostles, in their mother-tongue?
 But if not, why do you not show us where Christ or his apostles have forbidden it?
 
-But what do you say of the sacrament of the altar, as it is called?
+[.numbered]
+6+++.+++ But what do you say of the sacrament of the altar, as it is called?
```

Comment:
So #6 here was missing in the original PDF so I put it in where it seemed logical to me, but I could totally be off!

Reply:
Seems right to me.